.github/workflows/scripts/publish_client_gem.sh

```diff
@@ -3,37 +3,12 @@
 # WARNING: DO NOT EDIT!
 #
 # This file was generated by plugin_template, and is managed by it. Please use
-# './plugin-template --travis pulp_npm' to update this file.
+# './plugin-template --github pulp_npm' to update this file.
 #
 # For more info visit https://github.com/pulp/plugin_template
 
 set -euv
 
-cd $GITHUB_WORKSPACE/../pulp-openapi-generator
-COMMIT_MSG=$(git log --format=%B --no-merges -1)
-export PULP_BINDINGS_PR_NUMBER=$(echo $COMMIT_MSG | grep -oP 'Required\ PR:\ https\:\/\/github\.com\/pulp\/pulp-openapi-generator\/pull\/(\d+)' | awk -F'/' '{print $7}')
-
-if [ -n "$PULP_BINDINGS_PR_NUMBER" ]; then
-  git fetch origin pull/$PULP_BINDINGS_PR_NUMBER/head:$PULP_BINDINGS_PR_NUMBER
-  git checkout $PULP_BINDINGS_PR_NUMBER
-fi
-
-./generate.sh pulpcore ruby 0
-cd pulpcore-client
-gem build pulpcore_client
-gem install --both ./pulpcore_client-0.gem
-cd ..
-
-./generate.sh pulp_npm ruby 0
-
-cd pulp_npm-client
-gem build pulp_npm_client
-gem install --both ./pulp_npm_client-0.gem
-cd ..
-
-sudo rm -rf ./pulpcore-client
-sudo rm -rf ./pulp_npm-client
```

Comment:
I believe we need this

Reply:
It's not present in the current template, I didn't manually delete it.